homework/english/Vladimir_Shelest/lecturer_2/Homework_Grammar.md

```diff
@@ -0,0 +1,68 @@
+Упражнение 1. Put the verbs in brackets in the Past Simple or in the Present Perfect.
+
+    1) I (have, just) ______ a nice pot of coffee. Would you like a cup?
+    2) I (see, not) ______ Steve this morning yet.
+    3) Carol and I are old friends. I (know) ______ her since I (be) ______ a freshman in high school.
+    4) Maria (have) ______ a lot of problems since she (come) ______ to this country.
+    5) I (go) ______ to Paris in 2003 and 2006.
+    6) A car came round the corner and I (jump) ______ out of the way.
+    7) Don’t throw the paper away because I (not to read) ______ it yet.
+    8) Is Jim going to eat lunch with us today? — No. He (eat) ______ (already).He (eat) ______ lunch an hour ago.
+    9) Since we (start) ______ doing this exercise, we (complete) ______ some sentences.
+    10) I (be) ______ never to Italy.
+
+    1) I've just had a nice pot of coffee. Would you like a cup?
+    2) I've not seen Steve this morning yet.
+    3) Carol and I are old friends. I've known her since I was a freshman in high school.
+    4) Maria has had a lot of problems since she came to this country.
+    5) I went to Paris in 2003 and 2006.
+    6) A car came round the corner and I jumped out of the way.
+    7) Don’t throw the paper away because I've not read it yet.
+    8) Is Jim going to eat lunch with us today? — No. He's already eaten.He ate lunch an hour ago.
+    9) Since we started doing this exercise, we have completed some sentences.
+    10) I've never been to Italy.
+
+Упражнение 2. Put the verbs in brackets in the Past Simple or in the Present Perfect.
+
+    1) You (ever work) ______ in a shop?
+    2) I (work) ______ at my uncle’s shop when I was younger.
+    3) It’s the first time I (be) ______ on a ship.
+    4) Ann is looking for her key. She can’t find it. She (lose) ______ her key.
+    5) How many symphonies Beethoven (compose) ______?
+    6) Look! Somebody (spill) ______ ink on the notebook.
+    7) You (have) ______ a holiday this year yet?
+    8) You (see) ______ any good films recently?
+    9) He (have, not) ______ any problems since he (come ) ______ here.
+
+    1) Have you worked in a shop?
+    2) I worked at my uncle’s shop when I was younger.
+    3) It’s the first time I was on a ship.
```

Comment:
It’s the first time I've been on a ship.
this is an experience, then Present Perfect.